src/vs/editor/browser/viewParts/viewCursors/viewCursor.ts

```diff
@@ -119,6 +122,8 @@ export class ViewCursor {
 	}
 
 	private _prepareRender(ctx: RenderingContext): ViewCursorRenderData {
+		let textContent = '';
+		const lineContent = this._context.model.getLineContent(this._position.lineNumber);
```

Comment:
I would prefer to put this expression still in if blocks to avoid unnecessary line content read

Reply:
Done